GameServer/gameutils/ChatUtil.cs

```diff
@@ -53,6 +29,822 @@ namespace DOL.GS
 	/// </summary>
 	public static class ChatUtil
 	{
+		/// <summary>
+        /// Used to send translated messages
+        /// </summary>
+        /// <param name="type">Determines the eChatType and eChatLoc to use for the message.</param>
+        /// <param name="target">The target client receiving the message (e.g., "client")</param>
+        /// <param name="translationID">The translation ID for the message (e.g., "AdminCommands.Command.Err.NoPlayerFound")</param>
+        /// <param name="args">Any argument values to include in the message, such as "client.Player.Name" (if no args, then use "null")</param>
+        /// <note>Please paste the translation ID's associated message (English) into a comment above this method.</note>
+        /// <example>ChatUtil.SendTypeMessage("cmdheader", client, "AdminCommands.Account.Description", null);</example>
+        /// <comment>To perform message changes, reference the '.txt' files located in 'GameServer > language > EN'. Please remember, if you change the translation ID value, check all other language folders (e.g., DE, FR, ES, etc.) to ensure all translated strings correctly reflect the new ID.</comment>
+        public static void SendTypeMessage(int type, GamePlayer target, string translationID, params object[] args)

```

Comment:
Alternatively, if a `eMsg` value is always associated to the same `eChatType` and `eChatLoc` values, and if the logic is always the same, a lookup table could be used instead of a switch statement.

```
// There are better ways to initialize an ImmutableDicitonary but they involve creating a helper class
// https://stackoverflow.com/questions/24033629/how-can-i-create-a-new-instance-of-immutabledictionary
private static ImmutableDictionary<eMsg, (eChatType, eChatLoc)> table = new Dictionary<eMsg, (eChatType, eChatLoc)>
{
    { eMsg.Advice, (eChatType.CT_Staff, eChatLoc.CL_SystemWindow) },
    { eMsg.Alliance, (eChatType.CT_Alliance, eChatLoc.CL_ChatWindow) },
    // And so on
}.ToImmutableDictionary();

// Usage
if (table.TryGetValue(type, out var result))
    target.Client.Out.SendMessage(translatedMsg, result.Item1, result.Item2);
```